packages/nimble-components/src/text-area/styles.ts

```diff
@@ -107,6 +120,47 @@ export const styles = css`
     :host([resize='vertical']) .control {
         resize: vertical;
     }
+
+    .container::after {
+        content: ' ';
+        position: absolute;
+        bottom: calc(-1 * ${borderWidth});
+        width: 0px;
+        height: 0px;
+        border-bottom: ${borderHoverColor}
+            var(--ni-private-hover-indicator-width) solid;
+        transition: width ${smallDelay} ease-in;
+    }
+
+    @media (prefers-reduced-motion) {
+        .container::after {
+            transition-duration: 0s;
+        }
+    }
+
+    :host([error-visible]) .container::after {
+        border-bottom-color: ${failColor};
+    }
+
+    :host(:hover) .container::after {
+        width: 100%;
+    }
+
+    :host([disabled]:hover) .container::after,
+    :host([readonly]:hover) .container::after {
+        width: 0px;
+    }
```

Comment:
put these in DOM tree order, so all the `.container` targetted styles should be before the `.control` targetted styles
```
:host
.label
.container.
.control
.error-icon
```

Reply:
done